from ..exceptions import RingError


class PolynomialError(RingError):
    pass


class PolynomialEvaluationError(PolynomialError):
    pass
